pkg/scheduler/internal/cache/node_tree.go

```diff
@@ -59,7 +59,8 @@ func (na *nodeArray) next() (nodeName string, exhausted bool) {
 	return nodeName, false
 }
 
-func newNodeTree(nodes []*v1.Node) *NodeTree {
+// NewNodeTree creates a NodeTree from nodes.
+func NewNodeTree(nodes []*v1.Node) *NodeTree {
```

Comment:
Why are we exporting this? Looks like we are using it only in internal/cache.

Reply:
Done. Thanks.